Content.Server/Forensics/Systems/ForensicPadSystem.cs

```diff
@@ -56,12 +56,20 @@ private void OnAfterInteract(EntityUid uid, ForensicPadComponent component, Afte
                 return;
             }
 
-            if (_inventory.TryGetSlotEntity(args.Target.Value, "gloves", out var gloves))
+            // Check access to fingerprints
+            var enumerator = _inventory.GetSlotEnumerator(args.Target.Value);
```

Comment:
You should use an `IInventoryRelayEvent` for this! If you raise the event on an entity it'll also raise it on all its clothing as well. Not hard to add and using the event will make adding new stuff that interacts with this a lot easier. See: (https://github.com/space-wizards/space-station-14/pull/31147) for examples! (If you need more examples, take a look at all the subscribed events in [InventorySystem.Relay.cs](https://github.com/space-wizards/space-station-14/blob/1335a11446bdcd07416d638bbb19064242905c41/Content.Shared/Inventory/InventorySystem.Relay.cs).